AutoRunCustomTool/Options/OptionsPageGrid.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.VisualStudio.Shell;
+using System.ComponentModel;
+
+namespace ThomasLevesque.AutoRunCustomTool.Options
+{
+    public class OptionsPageGrid : DialogPage
+    {
+        [Category("AutoRunCustomTool")]
+        [DisplayName("Extensions")]
+        [Description("What extension(s) to run the custom tool on")]
+        public string ListenToExtension { get; set; }
+
+        [Category("AutoRunCustomTool")]
+        [DisplayName("File to run tool on")]
+        [Description("The file that we should trigger a custom tool on save")]
+        public string ToolToRun { get; set; }
```

Comment:
From what I understand, `ToolToRun` actually contains the name(s) of the file(s) on which the custom tool must be run?
The name is really misleading, this property should be renamed (maybe to something like `TargetFiles`?